### PR TITLE
[polyfill] Upgraded tfjs-backend-wasm version to 2.7.0.

### PR DESCRIPTION
**Pull request by BruceDai**
There's an error when running with upgraded tfjs-backend-wasm with both enabled SIMD and Threads flags. Please see detail info [here](https://github.com/intel/webml-polyfill/issues/1369#issuecomment-697225080).

### CODE REVIEW

File: src/nn/tfjs/TfjsModel.js [hunk 1]
```diff
@@ -33,18 +31,9 @@ export default class TfjsModel {
...
-                parts[parts.length - 1] = wasmPath;
-                return parts.join('/');
-              }
-            }
-            return '';
-          }
-          setWasmPath(null, {'tfjs-backend-wasm.wasm': _fixWasmPath(wasmPath)});
+          // Assume the wasm file is located in the same folder as webml-polyfill.js
+          const wasmPathPrefix = window.location.protocol + "//" + window.location.host + "/dist/";
+          setWasmPaths(wasmPathPrefix);
```
Comment (BruceDai) [inline src/nn/tfjs/TfjsModel.js:36]:
```js
setWasmPaths(wasmPathPrefix); 
```
equals to 
```js
setWasmPaths({
  'tfjs-backend-wasm.wasm': wasmPathPrefix + 'tfjs-backend-wasm.wasm",
  'tfjs-backend-wasm-simd.wasm':  wasmPathPrefix + 'tfjs-backend-wasm-simd.wasm",
  'tfjs-backend-wasm-threaded-simd.wasm': wasmPathPrefix + 'tfjs-backend-wasm-simd.wasm",
});
```
For simple, I invoked setWasmPaths function with prefix string parameter.

**Comment by BruceDai** [comment]
Performance data (ms) of MobileNet v2 TFLite on ICL Acer Swift F3 (CPU: i5-1035G4)


| Model       | Wasm (Plain)          | Wasm (SIMD)  |Wasm (SIMD+Threads)   | WebGL  |
| ------------- | ------------- | ----- |----- | ----- |
| MobileNet v2 TFLite      | 139.54  | 45.03 |24.73 | 35.74 |

Wasm (SIMD+Threads) speedups 5.6X than Wasm (Plain)  and 1.8X than Wasm (SIMD) .

**Review by huningxin — APPROVED** [review]
Looks good. Thanks much @BruceDai !

**Comment by ibelem** [comment]
Awesome, thanks!

**Pull request merged**